k8s-daemonset/k8s/hello-world-grpc-legacy.yaml

```diff
@@ -0,0 +1,99 @@
+---
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: hello
+spec:
+  replicas: 3
+  selector:
+    app: hello
+  template:
+    metadata:
+      labels:
+        app: hello
+    spec:
+      dnsPolicy: ClusterFirst
+      containers:
+      - name: service
+        image: buoyantio/helloworld:0.1.4
+        env:
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: POD_IP
+          valueFrom:
+            fieldRef:
+              fieldPath: status.podIP
+        - name: NS
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        command:
+        - "/bin/sh"
+        - "-c"
+        - "helloworld -addr=:7777 -text=Hello -target=$NODE_NAME:4140 -protocol=grpc"
```

Comment:
It looks like `NODE_NAME` is no longer set as an env var for this config, so this should be:

```
- "helloworld -addr=:7777 -text=Hello -target=`hostIP.sh`:4140 -protocol=grpc"
```